app/helpers/application_helper.rb

```diff
@@ -2,4 +2,12 @@ module ApplicationHelper
   def last_updated_date
     File.mtime(Rails.root.join('REVISION')).to_date rescue Date.today
   end
+
+  def ajax_enabled_for?(name)
+    %w(
+      energy-grants-calculator
+      calculate-employee-redundancy-pay
+      register-a-death
+    ).exclude?(name)
```

Comment:
Also, please keep in mind that we don't plan to maintain that list. It is only temporary (for 2-3 weeks). That knowledge might change your approach.